api/SWR/package/usePackageGetAll.js

```diff
@@ -6,7 +6,7 @@ const usePackageGetAll = () => {
 
   const options = useMemo(() => data.map((type) => ({ value: type.id, text: type.name })), [data]);
 
-  return { data, options };
+  return { data, options, packageGetAllLoading: !!data };
```

Comment:
Let's name it `isLoading` everywhere to be consistent. If needed it can be renamed in desctructuring